tests/foreman/ui/test_contenthost.py

```diff
@@ -726,13 +726,13 @@ def test_positive_virt_who_hypervisor_subscription_status(session, rhel7_content
         checkin_time1 = session.contenthost.search(provisioning_server)[0]['Last Checkin']
         result = rhel7_contenthost.run('service virt-who stop')
         if result.status != 0:
-            raise CLIFactoryError(f'Failed to stop the virt-who service:\n{result.stderr}')
+            raise CLIFactoryError(f'Failed to stop the virt-who service:\n{result.stderr[1]}')
         result = rhel7_contenthost.run('virt-who --one-shot')
         if result.status != 0:
-            raise CLIFactoryError(f'Failed when executing virt-who --one-shot:\n{result.stderr}')
+            raise CLIFactoryError(f'Failed when executing virt-who --one-shot:\n{result.stderr[1]}')
         result = rhel7_contenthost.run('service virt-who start')
         if result.status != 0:
-            raise CLIFactoryError(f'Failed to start the virt-who service:\n{result.stderr}')
+            raise CLIFactoryError(f'Failed to start the virt-who service:\n{result.stderr[1]}')
```

Comment:
`result.stderr[1]` can we not inherit/or this join the stderr[1] to give response as result.stderr ?  (non blocking)

Reply:
right now, stderr is a tuple, but that might change in the future.

Reply:
saw that it returns the tuple with (code_number, error_message) , not sure anyone use that code.